### ISSUE TITLE
Volume similarity has the wrong sign

### ISSUE THREAD

**Issue by jhlegarreta** [Jingnan-Jia/segmentation_metrics#30]
I have realized that the volume similarity metric has the opposite sign to what it should have with respect to the definition you are giving here:
https://github.com/Jingnan-Jia/segmentation_metrics/blob/master/README_appendix.md#explanication-of-volume-similarity

A simple test case is enlightening to see this:
```
gdth_img = np.asarray([[0, 1, 0], [1, 1, 1], [0, 1, 0]])
pred_img = np.asarray([[1, 1, 0], [1, 1, 1], [0, 1, 0]])
spacing = np.ones((2,))
labels = [1, ]
import seg_metrics.seg_metrics as sg
metrics = sg.write_metrics(
    labels=labels,
    gdth_img=gdth_img,
    pred_img=pred_img,
    spacing=spacing,
)
assert metrics[0]["vs"][0] > 0
```

@Jingnan-Jia you will agree with me that in the above case, and following the second definition adopted by this package, VS should be positive in this case (i.e. the predicted volume is larger than the ground truth). The assertion raises an error, and thus, the implementation does not match the definition. 

The bug can be fixed by changing this:
https://github.com/Jingnan-Jia/segmentation_metrics/blob/71e15e66b42ef4181b40ce0d3a4ce0ee3645da38/seg_metrics/seg_metrics.py#L118

to
```
dicecomputer.Execute(labelPred > 0.5, labelTrue > 0.5)
```

It may be a good idea to check the rest of the package to see if other errors are present. Ideally, adding testing should allow to identify unexpected behavior, and ensure that results are consistent across versions.

Thanks for developing this package.

**Comment by Jingnan-Jia** [comment]
@jhlegarreta Thank you for pointing it out! I am checking and fixing this issue.

**Comment by Jingnan-Jia** [comment]
@jhlegarreta Thank you for your advice again. I updated the package and now you can get the correct volume similarity sign by updating the package to the latest version (>=1.2.8)
`pip install --upgrade seg-metrics`

Details of update: I removed the old code `dicecomputer.Execute(labelPred > 0.5, labelTrue > 0.5)` because it relied on ITK which was not explicit enough (the document of ITK for Python is not complete as well). I added the following code to explicitly calculate VS (also added the check of extreme conditions):
```
v_pred = np.sum(labelPred > 0.5)
v_gdth = np.sum(labelTrue > 0.5)
if v_pred + v_gdth == 0: # prediction results and ground truth are all 0
    vs = 0
else:
    vs = 2 * (v_pred - v_gdth) / (v_pred + v_gdth)
```

If you have more advice, please let me know.

**Comment by jhlegarreta** [comment]
@Jingnan-Jia sounds good. Thanks for the update. Closing the issue as resolved after bc37480.
